universal-application-tool-0.0.1/test/support/TestQuestionBank.java

```diff
@@ -143,6 +183,60 @@ private static Question applicantHouseholdMemberName(QuestionEnum ignore) {
     return maybeSave(definition);
   }
 
+  private static Question applicantPets(QuestionEnum ignore) {
+    QuestionDefinition definition =
+        new CheckboxQuestionDefinition(
+            VERSION,
+            "checkbox",
+            Path.create("applicant.checkbox"),
+            Optional.empty(),
+            "description",
+            LifecycleStage.ACTIVE,
+            ImmutableMap.of(Locale.US, "question?"),
+            ImmutableMap.of(Locale.US, "help text"),
+            ImmutableList.of(
+                QuestionOption.create(1L, ImmutableMap.of(Locale.US, "cat")),
+                QuestionOption.create(2L, ImmutableMap.of(Locale.US, "dog"))));
+    return maybeSave(definition);
+  }
+
+  private static Question applicantPronouns(QuestionEnum ignore) {
+    QuestionDefinition definition =
+        new DropdownQuestionDefinition(
+            VERSION,
+            "applicant preferred pronouns",
+            Path.create("applicant.preferred.pronouns"),
+            Optional.empty(),
+            "Allows the applicant to select a preferred pronoun",
+            LifecycleStage.ACTIVE,
+            ImmutableMap.of(Locale.US, "What is your preferred pronoun?"),
+            ImmutableMap.of(Locale.US, "help text"),
+            ImmutableList.of(
+                QuestionOption.create(1L, ImmutableMap.of(Locale.US, "He / him")),
+                QuestionOption.create(2L, ImmutableMap.of(Locale.US, "He / him")),
```

Comment:
(blocking): these options aren't right. 1L and 2L should be different `Locale.US`, and the `Locale.FRANCE` ones should also use 1L and 2L and be the french versions of the same thing.

Reply:
Sorry actually, it the `Locale.FRENCH` should be in the `ImmutableMap.of` along with the `Locale.US` localization.